.github/workflows/code_scan.yml

```diff
@@ -7,6 +7,28 @@ on:
     - cron: "0 18 * * 1-5"
 
 jobs:
+  Trivy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
```

Comment:
Is there any reason to set python version to `3.10` for trivy scanning?

Reply:
No, the python is required for installing datumaro package & its dependency, not for the Trivy. so any versions are okay if datumaro is supported.